src/platform_impl/windows/window.rs

```diff
@@ -616,9 +618,25 @@ impl Window {
         self.window_state.lock().taskbar_icon = taskbar_icon;
     }
 
+    pub(crate) fn set_ime_position_physical(&self, x: i32, y: i32) {
+        if unsafe { winuser::GetSystemMetrics(winuser::SM_DBCSENABLED) } != 0 {
```

Comment:
I installed an east asian language pack for testing and it only worked when switching to `SM_IMMENABLED` here.

Reply:
Sorry, I got mistake. Thanks,